### PR TITLE
Don't allow calls when the connection the server has been lost

### PR DESCRIPTION
**Pull request by SimonBrandner**
Type: enhancement
Fixes https://github.com/vector-im/element-web/issues/2096

<hr>

Screenshot_20211205_131119

---
Here's what your changelog entry will look like:

## ✨ Features
 * Don't allow calls when the connection the server has been lost ([\#7287](https://github.com/matrix-org/matrix-react-sdk/pull/7287)). Fixes vector-im/element-web#2096. Contributed by @SimonBrandner.

Preview: https://61adc42651bb255c5fbd5d04--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/CallHandler.tsx [hunk 1]
```diff
@@ -783,6 +784,14 @@ export default class CallHandler extends EventEmitter {
             return;
         }
 
+        if (MatrixClientPeg.get().getSyncState() === SyncState.Error) {
+            Modal.createTrackedDialog('Call Handler', 'Sync error', ErrorDialog, {
+                title: _t('Connectivity to the server has been lost'),
+                description: _t('You cannot place VoIP calls without a connection to the server.'),
```
Comment (t3chguy) [inline src/CallHandler.tsx:790]: wondering if VoIP is too jargon-y

File: src/CallHandler.tsx [hunk 2]
```diff
@@ -778,16 +778,16 @@ export default class CallHandler extends EventEmitter {
...
+                description: _t('You cannot place voice/video calls in this browser.'),
             });
             return;
         }
 
         if (MatrixClientPeg.get().getSyncState() === SyncState.Error) {
             Modal.createTrackedDialog('Call Handler', 'Sync error', ErrorDialog, {
                 title: _t('Connectivity to the server has been lost'),
-                description: _t('You cannot place VoIP calls without a connection to the server.'),
+                description: _t('You cannot place voice/video calls without a connection to the server.'),
```
Comment (t3chguy) [inline src/CallHandler.tsx:790]: does it need the voice/video specifier? Is calls not sufficient?

Comment (SimonBrandner) [reply src/CallHandler.tsx:790]: Probably not though I tend to use it just to be clear :man_shrugging:

**Review by t3chguy — APPROVED** [review]
Thanks!

**Pull request merged**